frontend/javascripts/oxalis/model/sagas/settings_saga.js

```diff
@@ -35,12 +37,22 @@ function* trackUserSettingsAsync(action: UpdateUserSettingAction): Saga<void> {
   }
 }
 
+function* showUserSettingToast(action: UpdateUserSettingAction): Saga<void> {
+  if (action.propertyName === "moveValue" || action.propertyName === "moveValue3d") {
+    // $FlowFixMe moveValue and moveValue3d are both numbers
+    const moveValue: number = yield* select(state => state.userConfiguration[action.propertyName]);
```

Comment:
Does extracting `action.propertyName` into a variable above the if help flow to understand that the value is of type number?

Reply:
Yes it does, nice, thanks!